CombineAIDriver.lua

```diff
@@ -97,8 +97,10 @@ function CombineAIDriver:init(vehicle)
 				self.pipe:setAnimationTime(self.pipe.animation.name, 1, true)
 			else
 				-- if there's no animation we have to use this, as seen in the Giants pipe code
-				self.pipe:setPipeState(CombineAIDriver.PIPE_STATE_OPEN)
-				self.pipe:updatePipeNodes(999999, nil)
+				--self.pipe:setPipeState(CombineAIDriver.PIPE_STATE_OPEN)
```

Comment:
Please don't comment out  unused code, remove it.

Reply:
I deleted the old code